tasks/main.yml

```diff
@@ -79,24 +79,6 @@
     mode: 0770
   become: "{{ deploydrupal_files_become }}"
 
-- name: clear caches.
-  shell: "{{ deploydrupal_drush_path }} cache-rebuild"
-  args:
-    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
-  when:
-    - not deploydrupal_site_install
-    - deploydrupal_drush_cache_rebuild
-
-# Drush ^9 uses the Drupal service container. We intentionally clear the Drupal
-# cache before the Drush cache to avoid fatal errors due to stale cache in the
-# service container.
-- name: clear drush cache.
-  shell: "{{ deploydrupal_drush_path }} cache-clear drush"
```

Comment:
Do we no longer need this?

```
$ drush cr -h
Rebuild a Drupal 8 site.

This is a copy of core/rebuild.php. Additionally
 it also clears Drush cache and Drupal's render cache.
```

This seems to indicate that it still does something. I think?

Additionally, tutorials for creating Drush commands with drush:9^ indicate that a `drush cr drush` is needed still from what I'm finding online.

Reply:
Or perhaps we don't need this since we have a `drush cr` in the `drush deploy` command now?

Reply:
[drush cr is not run by drush deploy](https://www.drush.org/deploycommand/). I will add this back in.